test/dryrun/context_vars/objectname_nsselector/policy.yaml

```diff
@@ -15,4 +15,4 @@ spec:
         kind: ConfigMap
         metadata:
           name: '{{ not (hasPrefix "inv" .ObjectName) | skipObject }}'
-        data: '{{ set .Object.data "hocus" "pocus" | toJSON | toLiteral }}'
+        data: '{{ set (.Object.data | default (dict)) "hocus" "pocus" | toJSON | toLiteral }}'
```

Comment:
Note: This is a workaround to handle unset `data` objects when `kubectl apply` leaves the `data` unset instead of initializing to `data: {}`. Prevents `error: assignment to entry in nil map`